exonum-java-binding-common/src/main/java/com/exonum/binding/common/message/BinaryTransactionMessage.java

```diff
@@ -88,10 +84,7 @@ public HashCode hash() {
   @Override
```

Comment:
>      // We can't use BB directly for hashing because rawTransaction#position might be changed
     // and it causes having different hashes for the same message.
// and it causes having different hashes for the same message.

This comment doesn't seem correct any longer.